src/App.tsx

```diff
@@ -1,26 +1,138 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { deleteTodo, getTodos, USER_ID } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import { FilterType } from './types/FilterType';
+import { Header } from './components/Header';
+import { ErrorInfo } from './components/ErrorInfo';
+import { Footer } from './components/Footer';
 
-const USER_ID = 0;
+function getFilteredTodos(todos: Todo[], filter: FilterType) {
+  const filteredTodos = [...todos];
+
+  switch (filter) {
+    case FilterType.all:
+      return filteredTodos;
+    case FilterType.active:
+      return filteredTodos.filter(todo => !todo.completed);
+    case FilterType.completed:
+      return filteredTodos.filter(todo => todo.completed);
+    default:
+      return filteredTodos;
+  }
+}
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [, setLoad] = useState(true);
+  const [error, setError] = useState('');
+  const [title, setTitle] = useState('');
+  const [filter, setFilter] = useState(FilterType.all);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [loadTodos, setLoadTodos] = useState<number[]>([]);
+
+  const filteredTodos = getFilteredTodos(todos, filter);
+
+  function getCountActive(): number {
```

Comment:
It is important to stick to one way of creating functions, fix it everywhere
```suggestion
  сonst getCountActive = (): number => {
```